### DISCUSSION TITLE
leafpad crashing under labwc 0.7.0 (segfault?)

### DISCUSSION THREAD

**Discussion by ludgie** [labwc/labwc#1444] [Q&A] [answered]
OS: arch linux
labwc 0.7.0
Virtualbox

Tried running leafpad and basically crashed
I have xorg-xwayland installed and the labwc build (AUR) has xwayland enabled
Crashes as well when trying to run xterm and chromium (xwayland)

Log with `labwc -s leafpad 2> labwc-log.txt`: https://paste.centos.org/view/raw/edd8f1e7

It basically crashes instantly and I see this
![image](https://github.com/labwc/labwc/assets/77110723/412e467a-02da-44c5-818e-7d338ace6d70)

Log with `labwc -d -s leafpad 2> log.txt`: https://paste.centos.org/view/raw/f8b17dd0
![image](https://github.com/labwc/labwc/assets/77110723/21a74f35-9fc0-455a-9c94-75cca2de9743)

**Comment by stefonarch** [comment]
Confirming with `xterm`, but got no backtrace.

**Reply by ludgie** [reply]
#1361 could be related

**Reply by ludgie** [reply]
Should I move this to an issue @stefonarch?

**Reply by ahesford** [reply, marked as answer]
Already done. #1445.
